charms/__init__.py

```diff
@@ -0,0 +1,2 @@
+from pkgutil import extend_path
```

Comment:
Not for Python 3.3 and after; in those, we could just leave out `__init__.py` entirely.  However, with Python 2.x and Python 3.x before 3.3, this work-around is required.  It is also future-proof with 3.3+, so it's better to include it unless you are making strict 3.3+ requirements for your lib.